### PR TITLE
fix: changed anchor tag to div for hamburger

### PR DESCRIPTION
**Pull request by johnchen383**
Issue: hamburger icon reloaded site on press
Solution: changed to div
Risk: not really

Reviewers @KennethZhu904

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -33,11 +33,11 @@ function Header() {
           <a href="/" className='header__logo'>
             <img src={phLogo} alt="logo" />
           </a>
-          <a href="/" className='header__toggle hide-for-desktop' onClick={toggleMenu}>
+          <div className='header__toggle hide-for-desktop' onClick={toggleMenu}>
             <span></span>
             <span></span>
             <span></span>
```
Comment (KennethZhu904) [inline src/components/Header.tsx:39]: What does this span do?

**Review by KennethZhu904 — CHANGES_REQUESTED** [review]
Good job, but I am unsure about this section of this code. Can you please have a look at it!

File: src/components/Header.tsx (hunk 1 above)
Comment (johnchen383) [reply src/components/Header.tsx:39]: It is the hamburger!

**Review by KennethZhu904 — APPROVED** [review]
LGTM!

**Pull request merged**